pkgs/test_core/lib/src/executable.dart

```diff
@@ -156,6 +156,9 @@ Future<void> _execute(List<String> args) async {
   } on FormatException catch (error) {
     stderr.writeln(error.message);
     exitCode = exit_codes.data;
+  } on FileSystemException {
+    stderr.writeln("Illegal filename");
```

Comment:
And then log it like so:

```suggestion
    stderr.writeln(loadException.toString(color: configuration.color));
```